manifest-minimal.yml

```diff
@@ -37,6 +37,9 @@ DISTGEN_RULES:
   - src: src/test/from-dockerfile/uwsgi.Dockerfile.tpl
     dest: test/from-dockerfile/uwsgi.Dockerfile.tpl
 
+  - src: src/test/pin-pipenv-version-test-app/environment
```

Comment:
One last nitpick - I'd keep the paths in sync - I mean - if the file is generated as `.s2i/environment`, I'd keep it in the .s2i folder in the source of the application as well.